src/main/java/org/humancellatlas/ingest/submission/SubmissionEnvelopeRepository.java

```diff
@@ -18,13 +20,16 @@
 @CrossOrigin
 public interface SubmissionEnvelopeRepository extends MongoRepository<SubmissionEnvelope, String> {
 
-    @RestResource(path="findByUuid", rel="findByUuid")
+    @RestResource(exported = false)
```

Comment:
Why do you _not_ want to export this? Isn't this potentially useful?

Reply:
It's only used internally in the core and requires the nested Uuid{UUID} object as a the search param

Reply:
Ok, that's good.  Would it therefore be worth exporting the `findbyUuidUUID` method with a cleaner name?

Reply:
It's exported below as "findByUuid", but does a "findByUuidUuid" under the hood

Reply:
Aha! Looking at the wrong view! Thanks 👍